src/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  geo_location_map = {
```

Comment:
Can we put this map in the `massdriver.yaml` as a type string with the `oneOf` pattern we used yesterday with the `taint`?

Reply:
Good idea, I'll work on this change now

Reply:
Actually, now that I looked at it again, I don't think switching it to a `oneOf` would work because what the map is doing is setting an actual Azure region (like `uksouth`) to the resource group for deployment based on the `data_location` the customer chooses for the Azure communication service.

Setting it up as the `title/const` wouldn't solve that problem.